tests/test_seed_behavior.py

```diff
@@ -8,32 +8,17 @@
 
 
 def test_seed_behavior():
-    # Test with seed=None
-    Platform.seed_everything(None)
+    # Test with a specific seed
+    Platform.seed_everything(42)
     random_value_1 = random.randint(0, 100)
     np_random_value_1 = np.random.randint(0, 100)
     torch_random_value_1 = torch.randint(0, 100, (1, )).item()
 
-    Platform.seed_everything(None)
+    Platform.seed_everything(42)
     random_value_2 = random.randint(0, 100)
     np_random_value_2 = np.random.randint(0, 100)
     torch_random_value_2 = torch.randint(0, 100, (1, )).item()
 
     assert random_value_1 != random_value_2

```

Comment:
just a quick question: IIUC, even it has the random seed (42), but it still will equal on random and bring some potential failed, isn't it?  or I missed something.

Reply:
oops, fixed.